src/wayland/dmabuf/mod.rs

```diff
@@ -286,6 +739,14 @@ pub enum ImportError {
     InvalidFormat,
 }
 
+/// Handler trait for [`DmabufFeedback`]
+pub trait DmabufFeedbackHandler {
+    /// This function allows to override the default [`DmabufFeedback`] for a surface
```

Comment:
What does a return value of `None` mean? Also how do we account for the surface here?

Reply:
I will clarify None, but it means to not override the default feedback which will then be sent.
And yes, I forgot to add the surface 😅

Reply:
done